packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -181,4 +190,103 @@ describe('<DataGrid />', () => {
       );
     });
   });
+  describe('column width', () => {
+    it('should set the columns width to 100px by default', () => {
+      const rows = [
+        {
+          id: 1,
+          username: 'John Doe',
+          age: 30,
+        },
+      ];
+
+      const columns = [
+        {
+          field: 'id',
+        },
+        {
+          field: 'name',
+        },
+        {
+          field: 'age',
+        },
+      ];
+
+      const { getAllByRole } = render(
+        <div style={{ width: 300, height: 300 }}>
+          <DataGrid columns={columns} rows={rows} />
+        </div>,
+      );
+
+      const DOMColumns = getAllByRole('columnheader');
+      DOMColumns.forEach((col) => {
```

Comment:
No need for this intermediary variable. We have obviously struggled to find a name 😁

```suggestion
      getAllByRole('columnheader').forEach((col) => {
```